.github/workflows/tests.yml

```diff
@@ -0,0 +1,45 @@
+name: 'Tests'
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - '**.md?'
+
+jobs:
+  lint:
+    name: 'Lint (node: 20)'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'strapi/blocks-react-renderer'
+          persist-credentials: false
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: 20
+      - name: 'Install dependencies for linting'
+        run: yarn install
+      - name: 'Run lint'
+        run: yarn lint
+
+  unit_tests:
+    name: 'unit_tests (node: ${{ matrix.node }})'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [18, 20]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'strapi/blocks-react-renderer'
+          persist-credentials: false
+      - uses: actions/setup-node@v2-beta
```

Comment:
i think this could be the latest version which might be 4? or maybe 3?

Reply:
thanks, it's v3, I also bumped the checkout action to v4